tests/test_s3bucket.py

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
I think we should additionally test basepath functionality

To see which lines are not covered in your tests, update these lines to setup.cfg
```
[coverage:run]
omit = tests/*, prefect_twilio/_version.py

[coverage:report]
fail_under = 80
show_missing = True
```

And then 
```
coverage run --branch -m pytest tests -vv && coverage report
```

Should return something like: the last column is the lines that are missing cov
<img width="1319" alt="image" src="https://user-images.githubusercontent.com/15331990/181592896-4f6c4e3c-5942-45d4-9158-8f21e801fcd1.png">